doc/classes/EditorInterface.xml

```diff
@@ -277,6 +277,15 @@
 				Selects the file, with the path provided by [param file], in the FileSystem dock.
 			</description>
 		</method>
+		<method name="set_current_feature_profile">
+			<return type="void" />
+			<param index="0" name="profile_name" type="String" />
+			<description>
+				Selects and activates the specified feature profile with the given [param profile_name]. Set [param profile_name] to an empty string to reset to the default feature profile.
+				A feature profile can be programmatically created via the [EditorFeatureProfile] class.
```

Comment:
```suggestion
				A feature profile can be programmatically created using the [EditorFeatureProfile] class.
```

"Via" would mean "through", when in fact this class itself represents a profile.